### ISSUE TITLE
Tokens invalid for (seemingly) no reason

### ISSUE THREAD

**Issue by DRBragg** [scambra/devise_invitable#804]
I'm using this gem in one of my apps and overall it works pretty well.  However I am having an abnormal amount of users getting "invalid token" errors when trying to accept their invites.  I have expirations turned off but the only thing they share in common is the invites are from about a month ago.  Other users from that time accepted their invites no problem but ones that waited are having issues.

Thoughts?

**Comment by scambra** [comment]
Only the last invitation token is valid, so is it possible these users were invited again and they are trying to accept first invitation?

**Comment by DRBragg** [comment]
That's not the issues here.  Only 1 invite has been sent out.

**Comment by martingjaldbaek** [comment]
I ran into the same issue. What I finally managed to figure out is that some email providers pre-click all links in all emails as a phishing/spam/malware protection. Unfortunately, that also includes clicking the 'Reject invitation' link I have in the invitation emails. I managed to fix this by changing the link in the email from directly rejecting, to going to a page from which the user then has to click a reject button (the email bots don't do that).
